packaging/README.md

```diff
@@ -31,7 +31,9 @@ The following commands assume that the CloudStack source is present in **/tmp/cl
 
 ``docker run -ti -v /tmp:/src ubuntu:14.04 /bin/bash -c "apt-get update && apt-get install -y dpkg-dev python debhelper openjdk-7-jdk genisoimage python-mysql.connector maven lsb-release devscripts && /src/cloudstack/packaging/build-deb.sh"``
 
-The commands above will generate Ubuntu 14.04 and 16.04 packages which you will find in */tmp* on your system after the build succeeds.
```

Comment:
@kiranchavala 
it would be better to use
```
docker run -ti -v /tmp:/src ubuntu:22.04 /bin/bash -c \
"apt-get update && \
apt-get install -y software-properties-common && \
apt-add-repository universe --yes \
...
```

Reply:
@weizhouapache added the suggestion

Reply:
oops, it does not work as expected.  sorry for it
see
https://github.com/apache/cloudstack/blob/2bb3bf00344173f3c39883d27faf4919de3512ad/packaging/README.md

Reply:
reverted back the changes